template/src/components/fetching/index.js

```diff
@@ -0,0 +1,141 @@
+// React Hot Reload does not support stateless function components as of now
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import { provideHooks } from 'redial';
+import isoFetch from 'isomorphic-fetch';
+
+import styles from './style.scss';
+
+/* functions that promise to fetch json data from GitHub API */
+function fetchRocRepos() {
+  return isoFetch('https://api.github.com/users/rocjs/repos')
+  .then(response => response.json());
+}
+
+function fetchRocStargazers() {
+  return isoFetch('https://api.github.com/repos/rocjs/roc')
+  .then(response => response.json());
+}
+
+/* components for simple presentation */
+class RepoList extends Component {
+  static propTypes = {
+    display: React.PropTypes.number,
+    loading: React.PropTypes.bool,
+    repo: React.PropTypes.arrayOf(React.PropTypes.object),
+  };
+
+  static defaultProps = {
+    display: 5,
+    repoItems: [],
+    loading: true,
+  };
+
+  render() {
+    if (this.props.loading) {
+      return <ul><li>Loading...</li></ul>;
+    }
+
+    let repoItems = [];
```

Comment:
What do you think about naming this something other than what the prop is called, maybe `repoItemsToDisplay` or similar.